pkg/autodiscovery/autoconfig.go

```diff
@@ -14,6 +14,7 @@ import (
 
 	log "github.com/cihub/seelog"
 
+	autodiscovery "github.com/DataDog/datadog-agent/pkg/autodiscovery/config"
```

Comment:
I'm mixed about the naming here, it feels weird to import in the`autoconfig` package a package named `autoconfig`.
`config` would conflict with the existing `config` package, `check` also but maybe we could go with something like `integration`? So that we don't have to rename while importing and `integration.Config` sounds OK

WDYT @olivielpeau ?

Reply:
agreed, this sounds weird. `integration.Config` sounds better to me too 👍